src/webapp/contacts_reader.py

```diff
@@ -31,6 +31,7 @@ def get_tree(self, authorized=False, filters=None) -> Optional[OrderedDict]:
             self.yaml_data["ContactInformation"]["PrimaryEmail"])
         tree["Profile"] = self.yaml_data.get("Profile", None)
         tree["GitHub"] = self.yaml_data.get("GitHub", None)
+        tree["CILogonID"] = self.yaml_data.get("CILogonID", None)
```

Comment:
This reminds me - I think at some point per discussion with you i made a `cilogon_id` property [here](https://github.com/opensciencegrid/topology/blob/6b9eb1d7/src/webapp/contacts_reader.py#L61-L63) rather than touching the yaml data directly.  Is that relevant for any of your changes?

Reply:
I think for use within the same class, the extra layer of indirection from the property isn't necessary; outside of User, I use the cilogon_id property.